.github/workflows/run-test-cases.yml

```diff
@@ -186,7 +186,7 @@ jobs:
       name: Install Kubernetes
       run: |
         sudo apt-get update -y
-        sudo apt-get install -y --allow-downgrades kubelet=${{ matrix.kube-version }} kubeadm=${{ matrix.kube-version }} kubectl=${{ matrix.kube-version }} 
+        sudo apt-get install -o Dpkg::Options::="--force-overwrite" -y --allow-downgrades kubelet=${{ matrix.kube.version }} kubeadm=${{ matrix.kube.version }} kubectl=${{ matrix.kube.version }} 
```

Comment:
i think if `kube.version` is changed back to `kube-version` this should work. Daz is also changing the tst matrix in this commit https://github.com/deislabs/akri/commit/2e168fdc641f5b7182491dc649debcafc202e7ee, which is why the `kube-version` variable was changed

Reply:
I tried it locally on a vm and it still fails with the same error in the workflow. Looking around to understand the error.

Reply:
or maybe not.. I was missing the last 00 in the version when I tried it locally. Hopefully it works now.